src/server/acl/acl_family_test.cc

```diff
@@ -64,22 +64,27 @@ TEST_F(AclFamilyTest, AclDelUser) {
   EXPECT_THAT(resp, ErrArg("ERR wrong number of arguments for 'acl deluser' command"));
 
   resp = Run({"ACL", "DELUSER", "default"});
-  EXPECT_THAT(resp, ErrArg("ERR The'default' user cannot be removed"));
+  EXPECT_THAT(resp, IntArg(0));
```

Comment:
I am a little bit obsessed with this, haha. 😄  The Redis behaviour is: whenever the `default` user appears in the argument list, the whole command doesn't do anything and returns `ERR The 'default' user cannot be removed`.